src/services/auth.ts

```diff
@@ -39,7 +39,9 @@ function checkAuth(req: Request, res: Response, next: NextFunction) {
         res.redirect('login');
         return;
     } else if (!req.session.loggedIn && !noAuthentication) {
-        const redirectToShare = options.getOptionBool("redirectBareDomain");
+
+        const redirectToShare = hasRedirectBareDomain();
```

Comment:
Try simply `getOptionOrNull("redirectBareDomain") === "true"`.
It's not perfect, but good enough.
We can implement a `getOptionBoolOrNull` at some point.

Reply:
done in 1979affa8a6232b704c3a967ed3e14ce1a172d63